service/src/test/java/bio/terra/user/app/controller/ProfileApiControllerTest.java

```diff
@@ -71,10 +75,44 @@ void multiUser() throws Exception {
     assertUserProfile("$.value.name", "John");
   }
 
+  @Test
+  void adminNoPermission() throws Exception {
+    when(samService.adminGetUserIdByEmail(any(), any())).thenThrow(new ForbiddenException(""));
+    mockMvc
```

Comment:
nit Write helper method. It makes test easier to read, and another test could use the method in the future.

```
  @Test
  void adminNoPermission() throws Exception {
      when(samService.adminGetUserIdByEmail(any(), any())).thenThrow(new ForbiddenException(""));
      setUserProfileExpectCode("name", "{ \"value\": \"John\" }", "user2@gmail.com", HttpStatus.SC_FORBIDDEN);
  }
```